src/app/pages/common/entity/table/expandable-table/expandable-table.component.ts

```diff
@@ -8,6 +11,7 @@ export interface InputExpandableTableConf extends InputTableConf {
   expandedIfNotEmpty?: boolean;
   detailsHref?: string;
   limitRows?: number;
+  serviceStatus?: ServiceStatus;
```

Comment:
I think service status does not belong to a generic expandable-table component.
I think @RehanY147 had some ideas on how to implement this in a more generic way.

Reply:
What I had in mind was instead of adding `service-status` we can possibly give the header row's control to the parent component through `ng-content`, with maybe a `customActionsRow` flag or something like that.

Reply:
Good idea! Did some refactoring